### PR TITLE
Added support for chargers that are not reachable as installations

### PR DESCRIPTION
**Pull request by mrfrostfire**
This adds support for PRO chargers that does not turn up as installations, and also removes some deprecation-warnings

### CODE REVIEW

File: custom_components/zaptec/switch.py [hunk 1]
```diff
@@ -17,13 +17,13 @@ async def async_setup_platform(
     hass: HomeAssistantType, config: ConfigType, async_add_entities, discovery_info=None
 ) -> bool:  # pylint: disable=unused-argument
     """Setup switch platform."""
-    api = hass.data.get(DOMAIN, {}).get("api")
-    if api is None:
+    acc = hass.data[DOMAIN]["api"]
```
Comment (Hellowlol) [inline custom_components/zaptec/switch.py]: Use .get like I did before. This will raise and exception and not return None

**Comment by mrfrostfire** [comment]
Ok i’ll change that back. I changed it to this because In anorher file you did it this way

File: custom_components/zaptec/switch.py (hunk 1 above)
Comment (mrfrostfire) [reply custom_components/zaptec/switch.py]: fixed

**Comment by mrfrostfire** [comment]
Thanks :)

**Comment by Olen** [comment]
Any chance of getting this in a new release?
Anyone verified it with HOME chargers?

I have tested the patch on our PRO-chargers and it works fine for me.

**Comment by Hellowlol** [comment]
I haven’t had time to test it yet. I’ll try to get it tested and released this weekend

**Pull request merged**